frontend/src/scenes/groups/RelatedGroups.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react'
+import { useActions, useValues } from 'kea'
+import { LemonTable, LemonTableColumns } from 'lib/components/LemonTable/LemonTable'
+import { RelatedActor } from '~/types'
+import { Skeleton } from 'antd'
+import { groupsModel } from '~/models/groupsModel'
+import UserOutlined from '@ant-design/icons/lib/icons/UserOutlined'
+import { capitalizeFirstLetter } from 'lib/utils'
+import { urls } from 'scenes/urls'
+import { Link } from 'lib/components/Link'
+import { asDisplay } from 'scenes/persons/PersonHeader'
+import { relatedGroupsLogic } from 'scenes/groups/relatedGroupsLogic'
+
+interface Props {
+    groupTypeIndex: number | null
+    id: string
+}
+
+export function RelatedGroups({ groupTypeIndex, id }: Props): JSX.Element {
+    const { relatedActors, relatedActorsLoading } = useValues(relatedGroupsLogic)
+    const { loadRelatedActors } = useActions(relatedGroupsLogic)
+    const { groupTypes } = useValues(groupsModel)
+
+    useEffect(() => {
```

Comment:
Q: Why not put this inside the logic as well?

Reply:
The issue I had was I didn't have access to these two in the logic. But i could make this into a prop, that's right! :)

Reply:
Solved